lib/Models/Catalog/Esri/ArcGisFeatureServerCatalogItem.ts

```diff
@@ -257,7 +256,9 @@ class FeatureServerStratum extends LoadableStratum(
         return undefined;
       }
 
+      // @ts-expect-error @types/proj4 doesn't define a constructor type
```

Comment:
I can see why the type checker might not be able to infer that the argument is a string for the next line. I just threw together #7344 to try to make it slightly less confused.

Reply:
Should be fixed by [this commit](https://github.com/TerriaJS/terriajs/pull/7311/commits/24af89ddb16849a9d8b74bab0480f7b1799f010d).